From 90e8b9066d877e040e791bbf206db0e5653e017a Mon Sep 17 00:00:00 2001
From: Daniel Veillard <veillard@redhat.com>
Date: Wed, 30 Jan 2013 17:31:37 +0100
Subject: [PATCH 02/14] Fix a couple of places where (f)printf parameters were
 broken

As reported by Thomas Jarosch <thomas.jarosch@intra2net.com>
---
 python/libxslt.c    | 10 +++++-----
 xsltproc/xsltproc.c |  2 +-
 2 files changed, 6 insertions(+), 6 deletions(-)

diff --git a/python/libxslt.c b/python/libxslt.c
index 6a4f1c3..8dd6c78 100644
--- a/python/libxslt.c
+++ b/python/libxslt.c
@@ -356,15 +356,15 @@ libxslt_xsltRegisterExtModuleElement(PyObject *self ATTRIBUTE_UNUSED,
     PyObject *pyobj_element_f;
     PyObject *pyobj_precomp_f;
 
-#ifdef DEBUG_EXTENSIONS
-    printf("libxslt_xsltRegisterExtModuleElement called\n",
-	   name, ns_uri);
-#endif
-
     if (!PyArg_ParseTuple(args, (char *)"szOO:registerExtModuleElement",
 		          &name, &ns_uri, &pyobj_precomp_f, &pyobj_element_f))
         return(NULL);
 
+#ifdef DEBUG_EXTENSIONS
+    printf("libxslt_xsltRegisterExtModuleElement called: %s %s\n",
+	   name, ns_uri);
+#endif
+
     if ((name == NULL) || (pyobj_element_f == NULL) || (pyobj_precomp_f == NULL)) {
 	py_retval = libxml_intWrap(-1);
 	return(py_retval);
diff --git a/xsltproc/xsltproc.c b/xsltproc/xsltproc.c
index 35f37e8..dfd6d31 100644
--- a/xsltproc/xsltproc.c
+++ b/xsltproc/xsltproc.c
@@ -319,7 +319,7 @@ static void endTimer(char *format, ...)
     va_start(ap, format);
     vfprintf(stderr,format,ap);
     va_end(ap);
-    fprintf(stderr, " was not timed\n", msec);
+    fprintf(stderr, " was not timed\n");
 #else
   /* We don't have gettimeofday, time or stdarg.h, what crazy world is
    * this ?!
-- 
1.8.4.1

